tests/all.tcl

```diff
@@ -2,21 +2,57 @@
 package require Memcache
 #load ./memcache[info sharedlibextension]
 
-memcache server add localhost 11211
```

Comment:
changing the tests to actually use tcltest might be a good thing to consider in the future, but doesn't need to be done now.